coalib/__init__.py

```diff
@@ -6,6 +6,10 @@
 
 
 def get_version():
+    '''The coalib package is a collection of various subpackages
```

Comment:
@sils1297 I hugely appreciate such a detailed explanation of the problem. I feel humbled. I squashed the commits into one as you told me so. As you suggested, to begin with I chose this newcomer issue but I have been following the tutorials and would love to contribute to coala in the long run. Thanks a lot.